src/modules/interaction.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-restricted-globals */
+
+function reset() {
+  localStorage.clear();
+  location.reload();
+}
+
+function clear() {
+  let checkList = [];
+  checkList = document.querySelectorAll('.check');
+  let taskList = JSON.parse(localStorage.getItem('taskList'));
+  checkList.forEach((element, index) => {
+    if (element.checked === true) {
+      taskList[index].completed = true;
+      localStorage.setItem('taskList', JSON.stringify(taskList));
+    }
+  });
+  taskList = taskList.filter((task) => task.completed === false);
+  taskList.forEach((task, id) => {
+    task.index = id + 1;
+  });
+  localStorage.setItem('taskList', JSON.stringify(taskList));
+  location.reload();
+}
```

Comment:
- [ ] The functionality of the `clear` and `update` functions doesn't work as expected. Watch this [video ](https://www.loom.com/share/0232fe099ee14207a46b87379547e319) to know more about the issue and kindly work on that.

Reply:
Thanks for the video, I would never find this bug if you didn't mention it. ❤️